### ISSUE TITLE
can this be used to play audio from http?

### ISSUE THREAD

**Issue by arfan** [JagarYousef/ChatVoicePlayer#11]
can this library be used to play audio from http/https source?

**Comment by ibrahim-iqbal** [comment]
Yes, I would also like to use this to play Audio from the serve via a URL.

**Comment by iranandroid** [comment]
Hi
is it possible play audio from http/https with this library?

**Comment by JagarYousef** [comment]
It should be since it passes the path to a [normal MediaPlayer](https://github.com/JagarYousef/ChatVoicePlayer/blob/2b98bd026a5d7dc5d3447ee3d1fa636497d94344/chatvoiceplayerlibrary/src/main/java/me/jagar/chatvoiceplayerlibrary/VoicePlayerView.java#L171)

I have not tested it
